src/shared-components/selectorButton/style.js

```diff
@@ -61,14 +61,18 @@ export const Radio = styled.div`
   transition: background-color ${ANIMATION.defaultTiming};
   width: ${SPACING.medium};
 
+  ${({ selector }) => selector === 'checkbox' && css`
```

Comment:
i'd move the other `border-radius` from L55 next to this so that it's clear it's being overwritten.

Actually probably better practice not to declare the same CSS attribute on the same class so maybe switch it out for a ternary based on the selector